content/news/playtest-20170303.md

```diff
@@ -0,0 +1,62 @@
+
+* HiDPI font rendering for players with high resolution displays.
+* Fixes for several multiplayer bugs that could stall or crash games when a player disconnects.
+* The Red Alert "shell map" background is now muted.
+* New behaviour for the Red Alert Gap Generator and Tiberian Dawn Obelisk.
+* Fixes for asset installation on 32 bit operating systems and from the original RA95 CDs.
+
+The Dune 2000 mod in particular has many changes to make it better match the gameplay and graphical polish of the original game, plus 6 new campaign missions.
+
+Check out [the full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/) for more details, and head over to the [download page](/download/) to try it for yourself!  Your feedback on these playtests is important, because they are the last chance to catch any serious bugs or undo any controversial balance changes before they are included in an official release.
+
+<div style="text-align:center" markdown="1">
+![High resolution font rendering](/images/news/20170303-gapgen.png)
+
+Red Alert Gap Generators now actively reduce enemy unit sight.
+</div>
+
+<div style="text-align:center" markdown="1">
+![High resolution font rendering](/images/news/20170303-hidpi.png)
+
+Players with high resolution laptop or 4k displays will appreciate crisper font rendering.
+</div>
+
+<div class="about-todo-divider"></div>
+
+The biggest changes in this playtest are behind the scenes; technical improvements that are not visible to players, but important for our development and modding community:
+
+* A new [trait conditions](https://github.com/OpenRA/OpenRA/wiki/Trait-Conditions) system provides much better control over trait interactions.
+* Actor vision has been overhauled, adding support for features like the RA active gap generators, D2K actors revealing themselves when attacking, and cliffs blocking vision in heightmap-enabled mods.
+* Simplified weapon definitions using inherited yaml templates.
+* Some initial steps towards removing code patterns that are not compatible with save game support.
+
+OpenRA grew out of the C&C modding community, and one of our major goals is to provide a flexible game engine that the community can use for their own projects.  Unfortunately, this goal has been hurt by OpenRA's single-engine approach, meaning that every new release would unavoidably break community mods.
+
+After many discussions, we have developed a new mod support strategy that we will implement over the next two OpenRA releases.  This playtest (and the subsequent final release) starts by adding support for parallel OpenRA installs, and have started developing a [mod template](https://github.com/OpenRA/OpenRAModTemplate) that includes instructions and scripts to simplify mod development and packaging.  Mods built using this template will be self contained games that function independently from our official mod installs.
```

Comment:
The second half of the middle sentence sounds weird. Maybe just add a "we", like ", and we have started developing a mod template"